index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+My experience has taught me that good code is code that is:
+
+1. Readable
+2. Testable
+3. Maintainable
+
+This seminar will justify why writing good code is important, before providing a _large toolkit of guidelines and techniques_ that can be applied to your code to make it more readable, testable and maintainable.
+
+This seminar can be adjusted by reviewing some of your existing code to see the kind of problems that your experiencing and focusing on the techniques that will apply to your code base. [Get in touch](mailto:hi@tkiley.co.uk) today to start learning how to write better code.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * A working definition of good code.
+ * How to make code readable.
+ * How to make code testable.
+ * How to make code maintainable.
+
+ Software & Technologies:
+
+* [Python](https://www.python.org/){:target="_blank"} - a popular programming language
 
 ### Code Review
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+A code review process involves having one developer review another developers changes to the source code before it is submitted. Doing this effectively can spot defects and share knowledge.
+
+**Having a code review process will simultaneously upskill your developers and improve the quality of your code.**
+
+Code review _ensures that the code understood by everyone_, as well as sharing coding best practices organically.
+
+This seminar will cover all the advantages of a code review process. It will explain how to implement one, and then how to make the most of it.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn about how code review can help your software development practices. This seminar can be customised to include reviewing someones code as part of a demonstration.
+
+#### Seminar Syllabus
+
+The seminar can cover:
+
+ * What a code review process is.
+ * What to look for in a code review.
+ * How to make your code easy to review.
+
+Software & Technologies:
+
+ * [Gitlab](https://about.gitlab.com/){:target="_blank"} - a widely used cloud tool for conducting code reviews
+
+As with all the other seminars, get in touch if you use a different platform for conducting code reviews and I can tailor the course to use that website to make it easier to apply this to your existing workflows.
```

Comment:
Add "want" to make it relevent to everyone? e.g. "if you want to use a different platform"